app/react/src/server/cra-config.js

```diff
@@ -1,10 +1,17 @@
 import fs from 'fs';
 import path from 'path';
 import semver from 'semver';
+import program from 'commander';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
-
 import { normalizeCondition } from 'webpack/lib/RuleSet';
 
+program
```

Comment:
Oh, that is not what I meant 😅 , sorry if it was out of context (I was on my phone)

[Here](https://github.com/storybooks/storybook/blob/next/app/react/src/server/framework-preset-cra.js#L4) is the preset entry poring to the webpack configuration for CRA. The second parameter that is passed is `options` (it was not in use in this preset)

You need to do something like this:

```js
export function webpackFinal(config, { configDir }) {
  if (!isReactScriptsInstalled()) {
    logger.info('=> Using base config because react-scripts is not installed.');
    return config;
  }

  logger.info('=> Loading create-react-app config.');

  return applyCRAWebpackConfig(config, configDir);
}
```

we can just pass the full `options` object in case there will be other needed props to use in future.

Reply:
Will update now, thanks!